Makefile

```diff
@@ -2,3 +2,9 @@
 
 test:
 	docker-compose run web sh scripts/run_tests.sh
+
+reset:
+	docker-compose down
+	docker-compose rm -vf
+	docker-compose build --no-cache
+	docker-compose up
```

Comment:
Nice, I always ended up running the docker-compose in another tab to get around the need for this but this is more elegant. Might consider calling this from the `test` function so that `make test` just works

Reply:
This just kills and deletes everything before rebuilding. You still gotta run the server in a separate tab. You can run it in the background with `docker-compose up -d`, then run `make test` in the same term. That's what the travis tests do. I like watching the server logs though.